libraries/eosiolib/contracts/eosio/map.hpp

```diff
@@ -157,6 +170,17 @@ namespace eosio::kv {
          name     payer;
       };
 
+      /**
+       * @ingroup keyvaluemap
+       * 
+       * @brief This struct represents the iterator for the `eosio::kv::map` data type.
+       * @details You will use the sets of functions and operations associated with this type to
+       * iterate through values in the map and reference them.
+       * 
+       * Note that this iterator type in `eosio::kv::map` is defined as `eosio::kv::map::iterator_t`.
+       * There is also a reverse iterator available, it is defined as `eosio::kv::map::reverse_iterator_t`.
+       * The only difference between iterator_t and reverse_iterator_t is the direction the ++ and -- operators go.
```

Comment:
The only difference between iterator_t and reverse_iterator_t is the direction the ++ and -- operators go. - would it be better to define what ++ / -- does for each iterator rather than just say they work in opposite ways?

Reply:
this note is at the iterator class level; I do not see a problem to have it here.
indeed the -- and ++ should be annotated as well separatelly.